keras_nlp/models/bert_test.py

```diff
@@ -22,6 +22,40 @@
 from keras_nlp.models import bert
 
 
+class BertPreprocessorTest(tf.test.TestCase):
+    def setUp(self):
+        self.vocab = ["[PAD]", "[UNK]", "[CLS]", "[SEP]", "[MASK]"]
```

Comment:
Is there a reason the `vocab` is defined sequentially like this?

Reply:
Just to avoid the case where the black formatter will stick one vocab element per line. I could go the simple list approach, but this will jump from 3 -> 15 lines of code.

There might also be some format annotations if we want, but I prefer to avoid those. No strong preference here.